Feat/etwreceiver performance increase (#20)
axoflow/opentelemetry-collector-contrib: receiver/etwreceiver/README.md

<!-- end autogenerated section -->

## Configuration
| Field                     | Default  | Description                                                                                     |
|---------------------------|----------|-------------------------------------------------------------------------------------------------|

> | `provider`                | required | Provider to be subscribed to                                                                    |

Reviewer: ```suggestion
| `provider`                | required | Provider to subscribe to                                                                    |
```

| `level`                   | optional | Log level of trace events to be included                                                        |
| `ignore_missing_provider` | false    | Allows the collector to continue working when the given provider is missing                     |

> | `buffer_size`             | -        | Kilobytes of buffer size allocated for each ETW trace session. Minimum is 4, maximum is 16384.* |

Reviewer: ```suggestion
| `buffer_size`             | -        | Buffer size allocated for each ETW trace session, in kilobytes. Minimum is 4, maximum is 16384.* |
```

| `buffer_size`             | -        | Buffer size allocated for each ETW trace session, in kilobytes. Minimum is 4, maximum is 16384.* |
| `minimum_buffers`         | -        | Minimum number of buffers allocated for each ETW trace session.*                                 |
| `maximum_buffers`         | -        | Maximum number of buffers allocated for each ETW trace session.*                                 |

> | `flush_timer`             | -        | How often, in seconds, any non-empty trace buffers are flushed.*                                 |

Reviewer 2: ```suggestion
| `flush_timer`             | -        | How often (in seconds), any non-empty trace buffers are flushed.*                                 |
```

Author: Would keep it this way, as it is 1:1 copy of the official MS docs.

| `event_buffer_size`       | 1000     | Number of ETW events the ETW receiver stores in memory for processing.                           |

> | `num_workers`             | 1        | Number of worker goroutines that process ETW events.                                             |

Reviewer 2: ```suggestion
| `num_workers`             | 1        | Number of worker goroutines spawned to process ETW events.                                             |
```
